README.md

```diff
@@ -78,6 +79,17 @@ To run just the scenarios locally, follow these steps:
     - Run `build-azure.ps1`. This will run all the scenarios under the [Tests](Tests) folder.
 8. Run `vagrant destroy -f` or the appropriate `cleanup-*.ps1` once you have finished to kill the virtual machine.
 
-Tests are written in [ServerSpec](serverspec.org), which is an infrastructure oriented layer over [RSpec](rspec.info).
+Each of the `build-*` scripts can take parameters at the command prompt. These are gnerally designed to tighten the feedback loop during development.
+
+| Parameter                     | Type      | Default Value    | Description |
+| ----------------------------- | --------- | ---------------- | -------------------------------------------- |
+| `-SkipPester`                 | Switch    | False            | Skips the Pester and PSScriptAnalyzer unit tests, going straight to the VM-based integration tests.             |
```

Comment:
Might want to emphasise that the integration tests are very slow but the pester ones are quick